lib/Paws/Net/Caller.pm

```diff
@@ -30,6 +31,7 @@ package Paws::Net::Caller;
         (defined $requestObj->content)?(content => $requestObj->content):(),
       }
     );
+    print STDERR Data::Dumper::Dumper($response);
```

Comment:
Left a Data::Dumper print in

Reply:
I think this has been fixed long ago as 42 does not show the behaviour.

If you are in doupt try this branch all the S3 commands are working there

https://github.com/byterock/aws-sdk-perl/tree/s3ObjectTagging

Reply:
I guess I was mistaken by this pull request still being open if it is supposedly fixed in 0.42 release? I don't know.
I cloned the `pplu/aws-sdk-perl` this morning and checked out `release/0.42` and I still run into URI encoding issues, namely related to the `=` symbol in S3 object keys. I guess I should open a separate ticket/pull request for that.

Reply:
That would be good 

you would help me alot if you ran it against the s3ObjectTagging branch as well.  
Though I do not think I have touched the 'Paws::Net::Caller'  class in my branch.

Any of the S3 commands that run though the RestXml  callers and responders have been fixed up.

Hopefully there will be a relase that will get things a little more stable.

Reply:
Ok so a doc or bucket key like this

TEST/year=2020/month=01/day=02/test.txt

well have a look and see what happens

not a very good name for a bucket  ;)  but that is not my call

Reply:
This is the recommended way to prefix object keys in order to get out of the box Hive-partitioning in S3.
https://docs.aws.amazon.com/glue/latest/dg/aws-glue-programming-etl-partitions.html